ledger/src/blockstore_db.rs

```diff
@@ -303,13 +302,29 @@ impl Rocks {
                         db_options.set_max_open_files(-1);
 
                         Rocks(
-                            DB::open_cf_as_secondary(&db_options, path, &secondary_path, names)?,
+                            DB::open_cf_as_secondary(
+                                &db_options,
+                                path,
+                                &secondary_path,
+                                cf_names.clone(),
+                            )?,
                             ActualAccessType::Secondary,
                         )
                     }
                 }
             }
         };
+        for cf_name in cf_names {
+            db.0.set_options_cf(
+                db.cf_handle(cf_name),
+                &[(
+                    "ttl",
+                    &std::env::var("SOLANA_ROCKSDB_COMPACTION_TTL")
```

Comment:
evil env! I definitely needs to plumb this.... I'm considering to introduce `RocksConfig` after validating this fix.